.travis.yml

```diff
@@ -26,22 +26,47 @@ jobs:
       script: HOME=/tmp coverage run --source=lbrynet -m twisted.trial --reactor=asyncio tests.unit
       after_success:
         - bash <(curl -s https://codecov.io/bash)
+
     - <<: *tests
-      name: "Unit Tests w/ Python 3.6"
-      python: "3.6"
+        name: "Unit Tests w/ Python 3.6"
+        python: "3.6"
```

Comment:
These should not be indented since they are at the same level as the fields being injected. `<<: *tests` is not a parent of `name` and `python` it's a sibling.